pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -17,12 +17,13 @@ import (
 	"testing"
 
 	"github.com/gorilla/mux"
-	"github.com/stretchr/testify/require"
 
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
 	mockprovider "github.com/hyperledger/aries-framework-go/pkg/internal/mock/provider"
 	"github.com/hyperledger/aries-framework-go/pkg/restapi/operation"
 	"github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange/models"
+
```

Comment:
Import should be grouped with the other third party imports

Reply:
done